### DISCUSSION TITLE
Five layer method implementation in Bodi

### DISCUSSION THREAD

**Discussion by prakashmurali** [viyomam/viyomam-discussions#2] [General]
In 2022-Q1 we plan to initiate our five layer method implementation. We are planning to start with a single 36 x 36 feet patch first. As part of this we have the following key steps:

- [x] Decide the list of plants and trees
- [x] Viyomam implementation plan **[in progress]**
- [x] Ground preparation work
- [x] Arranging for water source and plant protection
- [x] Plant procurement
- [x] Guru pooja and Bija arpana
- [x] Plantation
- [ ] Maintenance
- [ ] Harvest
- [ ] Looking back and lessons learned

We will be using this post to keep track of our progress.

**Comment by prakashmurali** [comment]
### Deciding the list of plants

We have two options as of now:
Plant names and counts assuming 36x36ft

**Option 1:** 
Guava 1
Mango 2
Murunga 16
Thuvarai 20
Banana 20
Sitaphal 22
Karuvep 36
Chilly 36

**Option 2:**
Guava 1
Mango 2
Cashew 3
Elakka 4
Avocado 4
Pista 4
Sapota 5
Sitaphal 9
Lemon 9
Pineapple 9
Murunga 9
Chilly 18
Karuvep 18
Manjal 18
Thuvarai 20
Banana 20

Option 1 is easier, so shall we go with it?
Please share your choice/vote in the replies. If you have other suggestions for plant lists, please post them also.

**Reply by sgmoorthy** [reply]
For first implementation we would go with option 1

**Reply by sgmoorthy** [reply]
|Plant name| in Tamil| Count|
|Guava | கொய்யா| 1 |
|Mango | மா மரம் |2 |
|Murunga| முருங்கை | 16 |
|Thuvarai | துவரை | 20 |
|Banana | வாழை |20 |
|Sitaphal |சீதாப்பழம் |22 |
|Karuvep | கருவேப்பிலை | 36 |
|Chilly | மிளகாய் | 36 |

**Reply by prakashmurali** [reply]
Adding the plan here for easy reference. I can make this in Tamil too if needed
[Model1Option1.pdf](https://github.com/viyomam/viyomam-discussions/files/7962454/Model1Option1.pdf)
.

**Comment by prakashmurali** [comment]
_(Reposting the messages and files from the Whatsapp channel)_

[5 Layer Plants & layout.pdf](https://github.com/viyomam/viyomam-discussions/files/7836024/5.Layer.Plants.layout.pdf)

Prashanth: Kindly find below a simple list of plants to plant in the Fiver layer ZBNF method. Please review this to help get the 5 layer process started in our Bodi farm. 

The above PDF has three slides
slide 1 shows the pictorial representation of the plants
slide 2 has the simple plant list table shown below
Slide 3 has the layout for planting
Detailed Plant list for Five Layer: https://docs.google.com/spreadsheets/d/1CPOT5nMq13a3-vpudexqVCXAL1uabdLzmiSq-oGsHgs/edit?usp=sharing 

Model 1 is this sheet is similar to Option 1 above.

**Comment by prakashmurali** [comment]
For the next set of tasks (ground prep, plant procurement etc.) shall we try out the workflow we thought about? 
One team comes up with the requirements and a fully money-based solution. The next team takes this and determine which Viyomam circle we want and how these tasks can be done by increasing the contribution of anbu and reducing the use of money. Overall we want to have something where we have at least 25% anbu contributions.

**Reply by sgmoorthy** [reply]
this sounds good to me, let make the workflow to follow. We want to get this implemented within Q1 so lets focus to get this done

**Reply by prakashmurali** [reply]
Just experimenting with this:
- [ ] Create both teams. Each team has at least 2 Viyomam volunteers.
- [ ]  Team 1 comes up with the requirements and money-based solution. They will talk to different people to understand the requirements and summarize in a message/doc. 
- [ ]  ​Team 2 will take these requirements and decide which circle to fall in based on overall progress goals (based on available time, volunteer resources, money and other on the ground factors). For each task they will decide how to implement (money, barter, service-oriented, anbu). 
- [ ] -Both teams will discuss and come to a conclusion
- [ ] -After this,  we select one volunteer who will be responsible for checking that each task is following the selected path (money/barter etc.). If not, discuss among both teams to check how to reorient the project to being it in line with the overall goals.

Overall, we can have a deadline of 15 days for this roadmapping? One week for team 1 and one week for team 2, followed a discussion day.

Does this sound good?

**Reply by prakashmurali** [reply]
I would like to be part of team 1. Who else wants to join me? :)

**Reply by sgmoorthy** [reply]
I can take part in team 2

**Reply by varunsivapithan** [reply]
Nataraja 🙏 Very good plan done by Prakash Sami🙏 Pl give a special permission for me to be in both the teams...

Pl consider the following:
1) Pl list out activities specifically to involve better as a team member.

2) Our high priority tasks need to be discussed prior before starting 

3) Day to day update on what work we have done for the activity this will help us be in the thought among us

4)Pl give a start date and end date too (just to have a drive)

5) Since we have Six members in this group.. we need to take commitment n participation within a day

Pl pardon me if any of the above is deviant... SIVA 🙏

**Reply by prashanthv1** [reply]
> I would like to be part of team 1. Who else wants to join me? :)

Sure, we can partner. Should be straight forward knowing the plants already.

**Reply by prakashmurali** [reply]
@varunsivapithan : Great suggestions!
@prashanthv1: Thanks

@varunsivapithan  Replying to your points:

1. The main tasks are:

For team 1:
- Make sheet with task details from beginning of land prep to end of planting
- Discuss tasks with Jayanth and make local enquiries
- Find cost estimates for each task and overall project
- Find time estimates for each task and overall project
- Share with Team 2

For team 2:
- Look at each item and see what is the best way to implement it based on Viyomam principles 
- In the sheet, classify each task as money/service-oriented/barter/anbu and give a description of how to achieve it
- Select one person to monitor the project progress to check if it stays in line with the principles

Finally, Team 1 and Team 2 meet and finalize (1) tasks (2) budget and (3) timeline of implementation.

2. High priority tasks:
I spoke to Jayanth yesterday and he mentioned that the fencing and water arrangement are the high priority activities. More details below.

4. Start today and end by 6th Feb (two weekends)? @sgmoorthy targets to have one 36x36 patch done by March 31st. So if we plan by early Feb, we get enough time to implement.

5. Done for now. Prashanth, Varun, Prakash in Team 1. Guru, Varun, Prakash in Team 2. We will get inputs from Jayanth as and when required.

**Comment by prakashmurali** [comment]
@prashanthv1 @varunsivapithan 
Here is a draft sheet with costs and implementation details for our upcoming tasks
https://docs.google.com/spreadsheets/d/1NsUyBVEsPGt58gvJPxVgpp69WCD2Xrli98sEElAE3aQ/edit?usp=sharing
Please feel free to edit and add anything we require.

(Cost for drip irrigation method and transport was obtained from Jayanth)

**Reply by sgmoorthy** [reply]
Sheet looks good, can we also try to capture following
1. Total required minimum maintenance period for 36x36
2. Rough cost for minimum maintenance duration
3. Proposed yield per month and 
4. Return of investment to close the outermost layer.

**Reply by prashanthv1** [reply]
1. Regular on-going maintenance will be required for Plants (watering, periodic nourishment)
2. This will continue until plants are fully grown which will vary from a few months to 3-5years depending on the plant, until they start flowering & bear vegetables/fruits
3. Yield - we had captured the plant, growing period & expected yield based on the variety. @prakashmurali - could we link that Plant document here & capture the yield info by plants in Option 1
4. ROI will be based on the Total investment / expected yield & market rate

**Reply by prakashmurali** [reply]
1. Yes, in the five layer model, we need regular maintenance - watering, jeevamritham, harvesting, deweeding, trimming etc. depending on the situations. This is not like Miyawaki, but the amount of maintenance isn't too much. 
To fix a minimum duration, we can keep the time frame as 3 years for the trees (demonstrate one fruiting cycle) and 1 year for the small plants (demonstrate one round of cultivation).
2. Rough cost for year 1 to year 8 is added to sheet
3. Yield per month - Instead of this, I have added yield per year, across 8 years. This is because trees wont yield for first 2-3 years depending on the variety
4. If we assume maintenance costs of 1 day per week labour, then most likely we can cover the initial implementation costs within 2 years.
In practice that may not be required, and we can cover the initial costs within a year itself.

Please see the new sheet "Yield details"in the same file

**Reply by prakashmurali** [reply]
Also, another aspect not covered in our sheets is that we can grow vegetables in the plot (layers 4 and 5) during the first 2 years or so (till the trees start shading the area). That brings some additional return on investment.
